tests/yast2_cmd/yast_nfs_client.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# All of cases is based on the reference:
+# https://www.suse.com/documentation/sles-15/singlehtml/book_sle_admin/book_sle_admin.html#id-1.3.3.6.13.6.21
+#
+# Summary: setup a nfs client
+#     Key Steps:
+#       - sets up a nfs service on localhost
+#       - adds, edits and deletes a mount point
+#       - restores all configs
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils qw(systemctl zypper_call);
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+    zypper_call("in nfs-client yast2-nfs-client nfs-kernel-server yast2-nfs-server", exitcode => [0, 102, 103, 106]);
+
+    # sets up a nfs service on localhost
+    my $servdir = script_output('mktemp -d');
+    assert_script_run("yast nfs-server start; yast nfs-server add mountpoint=$servdir");
+
+    my $tmpdir01 = script_output('mktemp -d');
+    my $tmpdir02 = script_output('mktemp -d');
+
+    # adds,edits and deletes a mount point and verify
+    assert_script_run("yast nfs add spec=localhost:$servdir file=$tmpdir01");
+    validate_script_output('yast nfs list 2>&1', sub { m%$tmpdir01% }, timeout => 90, proceed_on_failure => 1);
+    # nfs.service will run "mount -at nfs,nfs4" to mount all points on SLE12SP2~SP4, but NOT on SLE15+
```

Comment:
If on 15+ nfs.service does not mount nfs mounts then something else has to do it
There is no verification on 15+ without check of mounts

Reply:
I had thought this point, and I thought "mount" action belong to "nfs.service" but not "yast nfs-client",  so I didn't verify it on SLE15+. Yes, there must be verifications on SLE15+, it is robust.

Reply:
You could create function of the repeating code for verification